app/decorators/decidim/system/update_organization_decorator.rb

```diff
@@ -7,13 +7,13 @@ def self.decorate!
         Decidim::System::UpdateOrganization.class_eval do
           private
 
-          alias_method :original_save_organization, :save_organization
+          alias_method :previous_save_organization, :save_organization
 
           def save_organization
             # Customization for google_tag_manager
             organization.google_tag_manager_settings = form.google_tag_manager_settings
 
-            original_save_organization
+            previous_save_organization
```

Comment:
The problem here is that this nomenclature collides with these override in the trustedId module.

https://github.com/ConsorciAOC-PRJ/decidim-module-trusted-ids/blob/main/app/commands/concerns/decidim/trusted_ids/system/register_organization_override.rb
https://github.com/ConsorciAOC-PRJ/decidim-module-trusted-ids/blob/main/app/commands/concerns/decidim/trusted_ids/system/update_organization_override.rb
 
We have used precisely these names as are more semantic. We think is better to change it here, in the final application rather than the module. But if you want, I'll update the module to change in there.

Reply:
Organizations are overridden very often, I would use `trusted_ids_original_create_organization` in the module or `decidim_catalunya_original_create_organization` here with a comment

Reply:
ok done!